Formula/git-branchless.rb

```diff
@@ -0,0 +1,23 @@
+class GitBranchless < Formula
+  desc "High-velocity, monorepo-scale workflow for Git"
+  homepage "https://github.com/arxanas/git-branchless"
+  url "https://github.com/arxanas/git-branchless/archive/refs/tags/v0.3.7.tar.gz"
+  sha256 "d23341766c0f81fc5e0768a399139738d472bcdd315c62b2b5ac9ac857d0d501"
+  license "GPL-2.0-only"
+
+  depends_on "rust" => :build
```

Comment:
Looks like you need this.
```suggestion
  depends_on "rust" => :build
  depends_on "openssl@1.1"
```

Reply:
Yes, thank you! Surprised this didn't get flagged when I ran `brew test` locally 🤔